lib/Minify/HTML.php

```diff
@@ -97,32 +99,22 @@ public function process()
 
         // replace SCRIPTs (and minify) with placeholders
         $this->_html = preg_replace_callback(
-            '/(\\s*)<script(\\b[^>]*?>)([\\s\\S]*?)<\\/script>(\\s*)/i'
-            ,array($this, '_removeScriptCB')
-            ,$this->_html);
+            '/(\\s*)<script(\\b[^>]*?>)([\\s\\S]*?)<\\/script>(\\s*)/i', array($this, '_removeScriptCB'), $this->_html);
```

Comment:
i'll try disabling `method_argument_space`

actually i was looking for php-cs-fixer to solve the formatting issues that came up in previous PRs.